Protocols/index.md

```diff
@@ -100,4 +100,14 @@ The Integrated Rule-Oriented Data System (iRODS) is an open source data manageme
 
 # Connection Profiles
 
-[Connection profiles](../Cyberduck/Profiles) (`.cyberduckprofile`) are plugins describing specific connection settings for a hosting provider to make it easier to setup a connection to your provider. A connection profile is installed and adds a provider option in the protocol selection drop down menu in the *Connection* and *Bookmark* panels. No need to enter the connection details manually other than credentials.
\ No newline at end of file
+[Connection profiles](../Cyberduck/Profiles) (`.cyberduckprofile`) are plugins describing specific connection settings for a hosting provider to make it easier to setup a connection to your provider. A connection profile is installed and adds a provider option in the protocol selection drop down menu in the *Connection* and *Bookmark* panels. No need to enter the connection details manually other than credentials.
+
+## Preferences → Profiles
+
+```{note}
+Prelimitary documentation for Cyberduck 8 and Mountain Duck version 4.8
+```
+
+Select connection protocols to be installed in addition to the default protocols listed above. The connection profile will be installed after enabling the corresponding checkbox. To disable a connection profile simply uncheck the checkbox. The profile will be disabled after closing the application.
+
+**Note:** You can't disable the default protocols. If you disable the connection profile used in a bookmark the bookmark won't work after restarting the application.
```

Comment:
Set the sentence into the admonition note instead of using "**Note:**" as prefix.